bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Dynamics/Tabulated.cpp

```diff
@@ -81,10 +81,10 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Dynamics_Tabulated(pybind11::module&
             )
 
             .def(
-                "get_contribution_profile",
-                &Tabulated::getContributionProfile,
+                "access_contribution_profile",
```

Comment:
From our discussion, in python we are using the `get_*` keywords as we are not truly "accessing" the memory values by reference in python. If and when we implement that in the bindings we can have both get and access based methods if/where appropriate.

Reply:
I think its more confusing to have a different name for the python binding than for the cpp method.

And there are already a bunch of methods called `access_*` in python in OSTk already. So I think its best to keep the name consistent between cpp and python

Reply:
Moving the conversation to slack.